docs/walkthrough-1.7.md

```diff
@@ -0,0 +1,489 @@
+# Service Catalog Demonstration Walkthrough for Kubernetes v1.7.0 and Above
```

Comment:
I know we wanted a separate walkthrough, but I figured it would be for the older versions. Is it expected to look at this one or the existing one when first starting?